academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md

```diff
@@ -1326,21 +55,21 @@ MAKES requires data it indexes to be placed in a single JSON file, with each lin
     > [!TIP]
     > This tutorial uses Microsoft as an organization by default. You can target any organization by finding its NormalizedName in the Microsoft Academic Graph and then changing the value of the affiliationNormalizedNameFilter variable to said name.
     >
+    > You can also modify the script to create different subgraphs. See script comments in the Subgraph Creation section.
+    >
     > Please note that doing this may impact time estimates for script execution and index generation later on in this tutorial.
 
-5. Provide a **Job name**, change **AUs** to 50, and select **Submit**
+1. Provide a **Job name**, change **AUs** to 50, and select **Submit**
```

Comment:
>50 [](start = 45, length = 2)

not many people will have 50 AUs.
My job used 20 AUs and it ran for 56 min and costed $3.72, while the ADLA and AS are in the same Azure region.
Please try whether it is the case for you.